### ISSUE TITLE
Debugging an entire function app is no longer possible

### ISSUE THREAD

**Issue by wis3guy** [Azure/azure-functions-core-tools#1527]
### TLDR

1. The context menu to start a debug session on a Function **App** is no longer available. It would be really, *really* useful to have this back.
2. There seems to be a bug in the plugin that prevents patching the `host.json` in case no function names are defined in the debug configuration. As a result clearing the parameter value has no actual affect. See screenshot below. Imho it should always be patched to ensure it reflects the current debug configuration.

![Screenshot 2019-08-27 at 16 07 40](https://user-images.githubusercontent.com/6885003/63778299-d5686c80-c8e4-11e9-9eb2-3d026ba25a53.png)

### Context

When working with Durable Functions (an extension from Microsoft on top of Azure Functions) you need to start all functions that take part in a single orchestration. Typically you need at least 3 functions to be running, a starter, an orchestrator and a handler.

In previous versions of the plugin, within Rider 2019.1, i could simply debug a Function **App**. This ensured all functions in the app were running and i could debug my durable functions just fine. In the fore-last version of the plugin (3.26.0.528-2019.2) the context menu to run/debug a Function App was removed. A gutter icon was introduced to start only an individual function. As a result, i cannot debug durable functions anymore!

### Work around

The only work-around that i figured, was:

1. Start a debug session for any function in the Function App that you need to debug;
2. Stop the debug session;
3. Modify the debug configuration by clearing the function name field;
4. Modify the (patched) `.../bin/Debug/netcoreapp2.1/host.json` file, either removing the `functions` node entirely or setting its value to an empty array;
5. Start the debug session from the toolbar, with the modified configuration selected;

Obviously steps 1-4 only need to be done once.

**Comment by wis3guy** [comment]
Accidently created the issue in the wrong repo ... #fail (https://github.com/JetBrains/azure-tools-for-intellij/issues/238)
